homeassistant/components/laundrify/__init__.py

```diff
@@ -0,0 +1,62 @@
+"""The laundrify integration."""
+from __future__ import annotations
+
+from laundrify_aio import LaundrifyAPI
+from laundrify_aio.exceptions import ApiConnectionException, UnauthorizedException
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ACCESS_TOKEN, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+
+from .const import CONF_POLL_INTERVAL, DEFAULT_POLL_INTERVAL, DOMAIN
+from .coordinator import LaundrifyUpdateCoordinator
+
+PLATFORMS: list[Platform] = [Platform.BINARY_SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up laundrify from a config entry."""
+
+    session = async_create_clientsession(hass)
```

Comment:
any particular reason why you want to create a new client session instead of getting an existing one? Is there a concern of sharing the same client session for multiple config entries? If not then use `async_get_clientsession`, it will create a session if needed but it will be shared across all config entries